agents/contracts/bondingmanager/topics.go

```diff
@@ -0,0 +1,57 @@
+package bondingmanager
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+func init() {
+	// set topics
+	var err error
+
+	parsedBondingManager, err := abi.JSON(strings.NewReader(BondingManagerMetaData.ABI))
```

Comment:
The pattern is bearable, but yeah if there's a way to autogenerate all topics that would be great. IMO it would be less error-prone, and also potentially helpful for debugging where we can parse topics on a whim to inspect auxiliary events.